packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -956,7 +956,23 @@ void restoreVmServiceConnectFunction() {
   vmServiceConnectFunction = _waitAndConnect;
 }
 
+/// The JSON RPC 2 spec says that a notification from a client must not respond
+/// to the client. It's possible the client sent a notification as a "ping", but
+/// the service isn't set up yet to respond.
+bool _ignoreRpcError(dynamic error) {
```

Comment:
I'm missing a whole lot of context here, can you explain in comments why we can be sure that any error that satisfies these condition should be ignored?

Reply:
I'll expand this.